src/servers/src/http/event.rs

```diff
@@ -150,6 +144,37 @@ pub async fn add_pipeline(
         })
 }
 
+#[axum_macros::debug_handler]
+pub async fn delete_pipeline(
+    State(state): State<LogState>,
+    Extension(query_ctx): Extension<QueryContextRef>,
+    Query(query_params): Query<LogIngesterQueryParams>,
+    Path(pipeline_name): Path<String>,
+) -> Result<String> {
+    let handler = state.log_handler;
+    ensure!(
+        !pipeline_name.is_empty(),
+        InvalidParameterSnafu {
+            reason: "pipeline_name is required",
+        }
+    );
+
+    let version = query_params.version.context(InvalidParameterSnafu {
+        reason: "version is required",
+    })?;
+
+    let version = to_pipeline_version(Some(version)).context(PipelineSnafu)?;
+
+    handler
+        .delete_pipeline(&pipeline_name, version, query_ctx)
+        .await
+        .map(|_| "ok".to_string())
```

Comment:
I think returning a json might be better.

Reply:
We can either return `affected_rows` like normal SQL delete or return `{"name": "xxx", "version":"xxx"}` like `add_pipeline`, which one do you prefer?

Reply:
return `{"name": "xxx", "version":"xxx"}` like `add_pipeline`.

Reply:
It's better to keep consistency between our HTTP APIs. So please refactor these pipeline API responses with `code`, and `output` just like https://docs.greptime.com/user-guide/clients/http-api ?

Reply:
https://github.com/GreptimeTeam/greptimedb/blob/f9db5ff0d6d007e577287c90da3d74172596f735/src/servers/src/http/greptime_result_v1.rs#L31-L40

@killme2008 The return format is designed specifically for queries. However we can use a similar format for management api return values
```json
{
  "output": {
    "pipelines": {
      "version": "xxx",
      "name": "xx"
    }
  },
  "execution_time_ms": 31
}
```
What do you think?

Reply:
@shuiyisong Sound good. The query API doesn't need to be consistent with our API, It is preferable to be compatible with existing ecosystems.

Reply:
The latest commit introduce `GreptimedbManageResponse`. I intend to use this struct as all management api return format(currently for pipelines and scripts). The return body should look like following
```Json
{
  "pipelines": [
    {
      "name": "test_name",
      "version": "test_version"
    }
  ],
  "execution_time_ms": 42
}
```

PTAL @killme2008 @sunng87 @tisonkun @paomian